Tools/ardupilotwaf/boards.py

```diff
@@ -238,6 +238,10 @@ def configure_env(self, cfg, env):
             env.LIB += [
                 'winmm',
             ]
+            env.AP_LIBRARIES += [
```

Comment:
this should be only add on vehicle wscript. We get the fix comming for rover, it may need for antenna tracker too.